app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -47,11 +58,12 @@
  *
  */
 @ManagedUi(R.layout.screen_login)
-public class LoginActivity extends CommCareActivity<LoginActivity> {
-    
+public class LoginActivity extends CommCareActivity<LoginActivity> implements OnItemSelectedListener {
+
     public final static int MENU_DEMO = Menu.FIRST;
     public final static String NOTIFICATION_MESSAGE_LOGIN = "login_message";
     public static String ALREADY_LOGGED_IN = "la_loggedin";
+    public static String KEY_LAST_POSITION = "last_position_selected";

```

Comment:
Seems more reliable to store the id of the last app selected rather than its position in the list, but I don't feel all that strongly about it.